.github/workflows/update-lock-files.yml

```diff
@@ -34,6 +34,7 @@ jobs:
         run: |
           source build_tools/shared.sh
           source $CONDA/bin/activate
+          conda update -n base --all
```

Comment:
We do this instead of using a newer miniconda build because we rely on the conda environment/setup that already exists in GitHub runners?

Reply:
yes we use the runner conda, we could use setup-miniconda instead, but I thought sticking to what we have is is simpler for now.